packer/resources/features/elk-stack/install.sh

```diff
@@ -18,7 +18,18 @@ sleep 1
 ## Update index and install packages
 apt-get update
 apt-get --yes --force-yes install ruby ruby-dev logstash elasticsearch=1.4.4 \
-    nodejs python-pip golang
+    nodejs python-pip golang libwww-perl libdatetime-perl nginx
+
+## Install Cloudwatch monitoring scripts
+wget "http://aws-cloudwatch.s3.amazonaws.com/downloads/CloudWatchMonitoringScripts-1.2.1.zip" -O /tmp/CloudWatchMonitoringScripts.zip
+unzip /tmp/CloudWatchMonitoringScripts.zip -d /tmp/
+rm /tmp/CloudWatchMonitoringScripts.zip
+mv /tmp/aws-scripts-mon /usr/local/
+
+# Add disk space cron job
+crontab -u root - << EOM
+*/5 * * * * /usr/local/aws-scripts-mon/mon-put-instance-data.pl --disk-space-util --mem-util --disk-path=/ --disk-path=/data --auto-scaling --from-cron
+EOM
```

Comment:
I'm thinking that I might add it as a pre-cache feature only, but leave the cron job as an exercise for the reader. It's a bit specific to the config of the instances you're launching (e.g. disks and mounts) and whether or not you want to measure actual space (GB free) or utilisation (% full).

I went with utilisation here, but I know that you're using free space for media service.